src/main/resources/db/changelog/changelog20221025_2350.sql

```diff
@@ -0,0 +1,6 @@
+-- liquibase formatted sql
+
+-- changeset nika.avalishvili:1
+ALTER TABLE benefit
+ADD COLUMN "fk_ben_type_id" INT,
```

Comment:
I don't think we need to add this column
We should refer to the column with id itself by foreign key
More info: https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-foreign-key/

Reply:
I modified this section according to the guideline and solved all above mentioned issues too.

I'm just thinking, do we need to have separate tables which continuously adds new benefit_types and calculation_methods considering that we have only two types?  For now, with creation of new benefit we also create new records in benefit_type table and use additional memory.

Can we rearrange the scheme in such a way to not to add new row in benefit_type or calculation_method if it already exists? I will try to find solution.

Reply:
Yeah, I think we want to avoid creating additional rows for each benefit.
These tables should serve as a dictionary and they should always have 2 rows with possibility to add more via controller.
I suggest starting with looking into differences between `@ManyToOne`, `@OneToOne`, `@OneToMany` and `@ManyToMany`
Maybe we chose incorrect annotation for the relation